nursery/connect-to-wmi-namespace-via-wbemlocator.yml

```diff
@@ -0,0 +1,17 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: connect to WMI namespace via WbemLocator
+    namespace: collection/database/wmi
+    author: michael.hunhoff@fireeye.com
+    scope: function
+    att&ck:
+      - Execution::Windows Management Instrumentation [T1047]
+  features:
+    - and:
```

Comment:
maybe also look for (optional) string `ROOT\CIMV2`, `ROOT\DEFAULT`, and maybe others

Reply:
great idea! I've added the strings you suggested - we can add more as they are identified